### PR TITLE
set proper env variables for "istio-private/proxy" presubmits

### PR DESCRIPTION
**Pull request by clarketm**

### CODE REVIEW

**Review by howardjohn — APPROVED** [review]

**Comment by clarketm** [comment]
@fejta 
Can we get a quick review from you? Also, should/can we update the `CODEOWNERS` so `prow/config.yaml` is also owned by ` istio/wg-test-and-release-maintainers` (or should we move the `presets` to a new file)?

**Review by fejta — APPROVED** [review]

**Pull request merged**
